src/auth/test/auth.controller.spec.ts

```diff
@@ -4,21 +4,28 @@ import { Test, TestingModule } from '@nestjs/testing';
 import { AuthController } from './../auth.controller';
 import { UserType } from '../user_type.enum';
 import { faker } from '@faker-js/faker';
+import { SignInUserDto } from './../dto/signin_user.dto';
```

Comment:
A recommendation here, to avoid large import paths and improve your code reading is to use `@alias imports` by adding those to your `tsconfig.json`